packages/app/android/src/reactnative/java/io/invertase/firebase/common/SharedUtils.java

```diff
@@ -130,6 +130,23 @@ public static boolean isAppInForeground(Context context) {
     List<ActivityManager.RunningAppProcessInfo> appProcesses = activityManager.getRunningAppProcesses();
     if (appProcesses == null) return false;
 
+    // Check if current activity is a background activity
+    if(json.contains("background_activity_names"))
+    {
```

Comment:
suggest android-specific key name, nit - spacing

```suggestion
    if (json.contains("android_background_activity_names")) {
```